modules/drivers/camera/dag/camera.dag

```diff
@@ -28,74 +17,6 @@ module_config {
         config_file_path : "/apollo/modules/drivers/camera/conf/camera_front_12mm.pb.txt"
       }
     }
-    components {
```

Comment:
why remove this?

Reply:
1. No front fisheye installed on vehicle by default. 2. The `camera_compress` are not working as of now so I remove them by default, the `video_compress` is still functional properly under `video/dag/vide.dag`.